modules/accredited_representative_portal/app/services/poa_request_search_filter_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+DEFAULT_PARAMS = {
+  status: 'Pending',
+  sortField: 'ar_power_of_attorney_requests.created_at',
+  sortDirection: 'desc',
+  pageNumber: 0,
+  pageSize: 10
+}.freeze
+
+class PoaRequestSearchFilterService
+  def self.handle_filter(filter_params)
+    result = PoaRequestQueryParamsSchema.call(DEFAULT_PARAMS.merge(filter_params))
+    if result.success?
+      normalized_params = result.to_h
+      status = normalized_params[:status]
+      sort_field = normalized_params[:sortField]
+      sort_direction = normalized_params[:sortDirection]
+      page_number = normalized_params[:pageNumber]
+      page_size = normalized_params[:pageSize]
+
+      poa_requests = AccreditedRepresentativePortal::PowerOfAttorneyRequest.with_status(status)
+                                                                           .sorted_by(sort_field, sort_direction)
+                                                                           .paginated(page_number, page_size)
```

Comment:
It looks like we're already using the will_paginate gem hence the collision. Can you try using that instead of our custom scope here?